coroutines/src/commonTest/kotlin/io/github/ackeecz/ackeelities/coroutines/SingleCoroutineLauncherTest.kt

```diff
@@ -18,14 +18,20 @@ internal class SingleCoroutineLauncherTest : FunSpec({
 
     suspend fun suspendCoroutine() = suspendCancellableCoroutine<Unit> {}
 
-    test("new launch cancels previous launch") {
+    suspend fun trySuspendCoroutine(onCancellationException: () -> Unit) {
```

Comment:
what about just suspendCoroutine? find the "try" confusing, since it seems like you are just trying to suspend it but actually you suspend it and cancel it later.